litmus-portal/frontend/.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/node_modules/
+**/*.md
```

Comment:
Which .md file we ignoring here?

Reply:
ignoring for the eslint

Reply:
yes we are ignoring all md files for eslint

Reply:
ok